src/libs/actions/IOU.js

```diff
@@ -483,7 +483,10 @@ function getMoneyRequestInformation(
               (recentlyUsedPolicyCategory) => recentlyUsedPolicyCategory !== category,
           )
         : [];
-    const optimisticPolicyRecentlyUsedCategories = [category, ...uniquePolicyRecentlyUsedCategories];
+    const optimisticPolicyRecentlyUsedCategories = uniquePolicyRecentlyUsedCategories;
+    if (category) {
+        optimisticPolicyRecentlyUsedCategories.unshift(category);
+    }
```

Comment:
Let's add same fix for tags

Reply:
@s77rt Could you check this comment https://github.com/Expensify/App/issues/28517#issuecomment-1746071461 ? I can't test with Tag feature

Reply:
I think we can keep this as is for now.